storybook/stories/API/chart/SunburstChart.mdx

```diff
@@ -0,0 +1,16 @@
+import { Canvas, Meta, ArgTypes } from '@storybook/blocks';
+import * as SunburstStories from './SunburstChart.stories';
+
+# Sunburst
+
+The sunburst is a hierarchical chart, similar to a treemap, plotted in polar coordinates.
```

Comment:
A suggestion: It might be nice to write a bit more about the motivation, as to why you would want to use it. 
Given that you just spent some time implementing it, I am certain you have a good case ❤️